src/config_validator.py

```
@@ -0,0 +1,359 @@
+        'ConfigMap': ['apiVersion', 'kind', 'metadata', 'data'],
+        'Secret': ['apiVersion', 'kind', 'metadata', 'data'],
+        'PersistentVolume': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'PersistentVolumeClaim': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'Pod': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'Namespace': ['apiVersion', 'kind', 'metadata'],
+        'ReplicationController': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'Ingress': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'DaemonSet': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'StatefulSet': ['apiVersion', 'kind', 'metadata', 'spec'],
+        'Job': ['apiVersion', 'kind', 'metadata', 'spec'],
+    }
```

Ruff (0.13.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)